web/src/index.ejs

```diff
@@ -18,10 +18,10 @@
   <meta name="mobile-web-app-capable" content="yes">
 
 
-  <meta name="theme-color" content="<%= config.theme.colors.themeColor %>">
-  <meta name="msapplication-navbutton-color" content="<%= config.theme.colors.themeColor %>">
+  <meta name="theme-color" content="<%= config.lightTheme.colors.themeColor %>">
```

Comment:
Why does this file depend on the lightTheme? I think it should depend on `theme`

Reply:
There is no theme anymore. For native we are already using the naming `lightTheme` and `darkTheme` and I just adjusted this. So theme = lightTheme.

Reply:
First how do you enable the darkTheme in web (actually i've never seen it so far)? Secondly what happens if you use the dark theme? the configurations in the index.ejs are always set to the light theme. Does that have an impact on how the webapp looks like?

Reply:
Just an easteregg right now implemented by @Schedulaar quite some time ago. You can activate it by clicking on the footer in an opened city 5 times. The configurations in the index.ejs are just using the theme color which is the same for both themes. Could be changed sometime as well (bright and dark colors would be enough instead of duplicating the whole theme.

Reply:
Nice feature :) I think this is sth that should be communicated between new developers here. I was always wondering whats up with the different themes in web 😸 

> Could be changed sometime as well (bright and dark colors would be enough instead of duplicating the whole theme).

Agreed. But should be definitely an issue of future discussion.